advanced-cache.php

```diff
@@ -10,7 +10,16 @@
     exit;
 }
 
-$ce_dir = ( ( defined( 'WP_PLUGIN_DIR' ) ) ? WP_PLUGIN_DIR : WP_CONTENT_DIR . '/plugins' ) . '/cache-enabler';
+/*
+ * Set the CACHE_ENABLER_PLUGIN_DIR constant in your wp-config.php file if the plugin resides
+ * somewhere other than wp-content/plugins/cache-enabler/.
+ */
+if ( defined( 'CACHE_ENABLER_PLUGIN_DIR' ) ) {
```

Comment:
Assuming this is to specifically allow it to live in the MU Plugins?

Reply:
Indeed, otherwise it would always be looking in `wp-content/plugins/cache-enabler`.